observability-hub/otel-collector/otel-collector.yaml

```diff
@@ -56,11 +54,15 @@ spec:
       otlp:
         protocols:
           grpc:
-            auth:
-              authenticator: basicauth
+            tls: # TODO
+              cert_file: /path/to/server/cert.pem
+              key_file: /path/to/server/key.pem
+              client_ca_file: /path/to/client/ca.pem
```

Comment:
Ive to figure out if openshift offers them somewhere or can generate them for us.

Reply:
it does!! I'll get it...

Reply:
it's the service-ca-operator & done w/ annotations on configmap, i'll find it